homeassistant/components/mqtt/config_integration.py

```diff
@@ -56,10 +55,7 @@
 
 CONFIG_SCHEMA_BASE = vol.Schema(
     {
-        Platform.ALARM_CONTROL_PANEL.value: vol.All(
```

Comment:
Isn't this breaking since the user can no longer test specific options with dev tools check yaml config before restarting?

I'm not sure that's actually valuable though as the issue is way more helpful but they will only see it after they restart

Reply:
Good point. I'll add a not in breaking change section.

Reply:
done.